src/puppetlabs/trapperkeeper/services/webserver/jetty9_core.clj

```diff
@@ -546,6 +546,8 @@
                                maybe-logged)]
       (.setHandler s statistics-handler)
       (.setStopTimeout s (or shutdown-timeout default-graceful-stop-timeout))
+      (when-let [script (:post-config-script options)]
```

Comment:
Would it be better to invoke the script here or right before `.start` is called on the server in `start-webserver!`?  Even though that doesn't appear to be the case right now, just thinking that if we ever had some logic in between the end of `create-webserver` and the actual starting of the server, it would be good for the script to be executed at the last possible point that a configuration override would need to be made.